release-notes/opensearch-release-notes-1.3.0.md

```diff
@@ -0,0 +1,632 @@
+# OpenSearch and Dashboards 1.3.0 Release Notes
```

Comment:
Can we add release highlights for ml-commons?

Reply:
I just added a brief line saying ml-commons now becomes part of the release. Any other recommendations about what to say here?